README.md

```diff
@@ -2,6 +2,6 @@ Student: Derkach Ostap Group: CE 48 Subject: CSDT
 
```

Comment:
the main branch is not removed
develop is not a default branch
remove PR#1 if it is will be the main PR